docs/proposals/1199-multitenant-api-proposal/README.md

```diff
@@ -0,0 +1,248 @@
+  - Fair resource sharing
+  - Inter-tenant prioritization
+  - SLO attainment
+- Create a handle with which to attach scheduling policies, allowing for heterogenous scheduling behavior across tenants 
+  - Such as the [InferenceSLOPolicy](https://docs.google.com/document/d/1j2KRAT68_FYxq1iVzG0xVL-DHQhGVUZBqiM22Hd_0hc/edit?resourcekey=0-5cSovS8QcRQNYXj0_kRMiw&tab=t.0#heading=h.emkaixupvf39).
+- Detach identification from the modelName field
+
+## Usage
+
+The InferenceObjectives API surface has 3 general scopes:
+- Identification
+- Pool Binding
+- Policy Handle
+
+### Pool Binding
+
+Tackling the simplest first; the single Pool Binding field `PoolRef` is simply to associate a given InferenceObjectives object with a pool. Meaning that InferenceObjectives with duplicate ID's across different pools are considered valid.
+
+### Identification
+**Note**: This ID field is proposed for Phase 2, & Phase 1 will use the kube name as the identifier in the short term.
+
+The only field associated with identification is the `ID` field. A unique ID field was chosen (rather than using the metadata name), because:
+- We do not want to put the same restrictions on the string that is enfored on a kube resource name
+- The ID name may be duplicated across different pools
+- Use of a kube-generated name would force an upstream Auth mechanism to be aware of the `InferenceObjectives` API
+
+This ID can be any string; in the case of a MaaS platform, it may identify a customer via a one-way hashed JWT. Or for sharing an InferencePool between teams, it may be a team-id. 
+
+***Important***: In order to support a high volume of tenants, by default IGW will _allow_ unique IDs that do not have an explicit InferenceObjectives object defined. Instead the default for all policies will be used.
+
+#### Alternative consideration(s)
+- Expanding the PoolRef field to be plural was considered, however that was not selected to maintain simplicity. It is a decision that can be revisited in the future, however.
+
+### Policy Specification
+There are a few options with regards to how we handle policy binding. Some options that have been considered:
+
+- An `array` field on the `InferenceObjectives` which would contain `ObjectReferences`, allowing for binding of arbitrary policies
+  - Pros: highly flexible, allows user defined policy
+  - Cons: highly abstract & difficult to enforce meaningful defaults/limits on the policies allowed 
+- A one-of per policy field to either allow embedded configuration of a policy, or a reference to a policy
+  - Pros: flexibility of selection, reuse of policies
+  - Cons: reusability is only per policy, composite policy structure is still defined on the `InferenceObjectives`, deprecetion of experimental fields may leave an API with many old, deprecated fields, making for a messier API surface.
```

Comment:
I don't believe we will be adding experimental fields into the api, anything experimental will be done via annotations, once accepted it migrates to the api.

This is because once an api makes it to v1, we will not be able to remove fields from it without going to v2, and going from v1 to v2 is a massive effort that we don't generally want to do.